examples/mnist/quantized.cpp

```diff
@@ -1,124 +0,0 @@
-/*
```

Comment:
Is it necessary or used by something?
quantized.cpp doesn't appear in examples/CMakeLists.txt.
I suppose it is an old version of examples/mnist/train.cpp